static/src/javascripts/projects/facia/modules/ui/video-container.js

```diff
@@ -0,0 +1,75 @@
+/**
+    Video Container
+ */
+
+define([
+    'bean',
+    'common/utils/$',
+    'bootstraps/enhanced/media/video-player'
+], function (
+    bean,
+    $,
+    videojs
+) {
+
+    var containerPos = 0,
+        videoWidth = 700,
+        translateWidth = 0,
+        numberOfVideos;
+
+    function init() {
+        numberOfVideos = $('.js-video-playlist').attr('data-number-of-videos');
+        bindEvents();
+    }
+
+    function bindEvents() {
+        bean.on(document, 'click', $('.js-video-playlist-next'), function () {
+            moveCarousel('next');
+        }.bind(this));
+        bean.on(document, 'click', $('.js-video-playlist-prev'), function () {
+            moveCarousel('prev');
+        }.bind(this));
+    }
+
+    function moveCarousel(direction) {
+        $('.js-video-playlist').removeClass('video-playlist--end video-playlist--start');
```

Comment:
`$videoPlaylist = $('.js-video-playlist')` somewhere maybe? seems to be used a few times